python/cuml/linear_model/linear_regression.pyx

```diff
@@ -30,7 +30,7 @@ from libc.stdlib cimport calloc, malloc, free
 from cuml import Handle
```

Comment:
I know this isn't part of these changes, but as a side note, we should be using the handle from `pylibraft` everywhere and remove the cuml handle so we're not keeping duplicated wrappers around.